dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -165,13 +165,13 @@ cmd_start_check_postflight() {
 cmd_stop() {
   debug $FUNCNAME
   FORCE_STOP=false
-  if [[ "$@" == *"--force"* ]]; then
+  if [[ "$@" == *"--skip:graceful"* ]]; then
   	FORCE_STOP=true
   fi
 
   if [[ ${FORCE_STOP} = "false" ]]; then
-    info "Waiting for graceful stop of services..."
-    cmd_action "graceful-stop" "$@"
+    info "stop" "Stopping workspaces..."
+    cmd_action "graceful-stop" "$@" >> "${LOGS}" 2>&1 || true
```

Comment:
or just fail, the action would have reported the error
BTW I don't think we should hide report of graceful stop by default as it may take a while you may want to know what is performed (as in any long task)
--silent mode could do only the log but I think I would be interested in how many workspaces have been stopped and how many I need